### PR TITLE
Add RHEL 8.4 support

### PR DESCRIPTION
**Pull request by jkozol**
Official Guest Images for RHEL 8.4 can now be built be osbuild-composer. The internal naming convention has changed in this PR. Prior RHEL 8 versions all fell under the `rhel-8` name. RHEL 8.4 uses the name `rhel-84` across osbuild-composer. 

This is opened as a draft PR since the UEFI boot tests are still a WIP.

### CODE REVIEW

File: schutzbot/deploy.sh [hunk 1]
```diff
@@ -71,6 +71,10 @@ if [[ $ID == rhel ]]; then
     greenprint "Setting up EPEL repository"
     # we need this for ansible and koji
     sudo dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+    # we need the fedora edk2-ovmf repo for uefi boot since the rhel one only inculdes secureboot firmware
+    greenprint "Setting up edk2-ovmf"
+    sudo dnf install -y https://kojipkgs.fedoraproject.org//packages/edk2/20200801stable/1.fc33/noarch/edk2-ovmf-20200801stable-1.fc33.noarch.rpm
```
[the remark this answers is not in the document]
Comment (larskarlitski) [reply schutzbot/deploy.sh:76]: Agreed. Thanks for figuring this out.

**Comment by jkozol** [comment]
@henrywang Do you know why the `run ubi8 image` test failed? I don't know anything about the `ubi8` image so I would really appreciate some help debugging this failure.

File: schutzbot/deploy.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkozol) [reply schutzbot/deploy.sh]: I have replaced this with the `virt-install` flag `--boot uefi,nvram_template=/usr/share/edk2/ovmf/OVMF_VARS.fd`

File: test/data/ansible/check_ostree.yaml [hunk 2]
```diff
@@ -277,6 +277,7 @@
     - name: run ubi8 image
       command: podman run ubi8-minimal:latest cat /etc/redhat-release
       register: podman_result
+      become: yes
```
Comment (jkozol) [inline test/data/ansible/check_ostree.yaml:280]: This test was failing due to a permissions error so I am having it run as root. @henrywang Is there any issue with doing this?

**Review by teg — APPROVED** [review]
Amazing work @jkozol! Thanks for figuring this all out :)

**Review by larskarlitski — APPROVED** [review]
> Amazing work @jkozol! Thanks for figuring this all out :)

💯 

Can only say the same. Really nice. Thanks!

**Review by ondrejbudai — APPROVED** [review]
AMAZING! Thanks!

**Pull request merged**